lua/possession/telescope.lua

```diff
@@ -94,7 +94,7 @@ function M.list(opts)
             prompt_title = 'Sessions',
             finder = get_finder(),
             sorter = conf.generic_sorter(opts),
-            previewer = session_previewer(opts),
+            previewer = config.telescope.show_previewer and session_previewer(opts) or false,
```

Comment:
I'd suggest changing `show_previewer` to `previewer` and using:
```lua
vim.F.if_nil(config.telescope.previewer, session_previewer(opts))
```
This way user could pass any custom previewer or `false` to disable previewer, but if left `nil` then the default previewer is used.